regression/cbmc/Overflow_Subtraction1/main.c

```diff
@@ -1,4 +1,4 @@
-#define INT_MIN (-1<<(sizeof(int)*8-1))
+#define INT_MIN (1U<<(sizeof(int)*8-1))
 
```

Comment:
This is now at least an unusual definition of INT_MIN

Reply:
Yet it has defined behaviour. Some discussion can also be found at https://blog.regehr.org/archives/738. Actual header files appear to be using `-INT_MAX-1`.

Reply:
Reverted/fixed in the latest revision.